Blast blast parser end hsp query frame
PDB disordered entity wrapper disordered select id
PDB abstract property map contains id
Entrez egquery
motifs jaspar motif str
PDB chain get unpacked list
UniProt record has inrec fieldvals
PopGen FDist read handle
GenBank feature consumer definition definition
PDB disordered entity wrapper set parent parent
Restriction ov3 compatible end cls batch
PDB mmtf structure decoder set bio assembly trans bio_assembly_index input_chain_indices input_transform
GA Repair ambiguous repair init ambig_finder num_ambiguous
Graphics GenomeDiagram circular drawer draw scale track
PDB dssp init model in_file dssp acc_array
motifs jaspar jaspar5 fetch latest version base_id
Graphics chromosome counts set scale segment_name scale
PDB pdbparser parse header_coords_trailer
Pathway system reactions
PopGen FDist split fdist monitor
GenBank insdc scanner feed misc lines consumer lines
SearchIO ExonerateIO get fragments phase frags
Align summary info get letter freqs residue_num all_records letters to_ignore
PDB is aa residue standard
GA organism lt other
Restriction one cut modify cls location
UniProt gpa iterator handle
Phylo tree mixin collapse all target
Phylo read file
Nexus replace parenthesized ambigs seq rev_ambig_values
Blast blast parser end parameters filter
core read and call while uhandle method
Blast blast parser end hsp gaps
NeuralNetwork Gene schema encode motif motif
SearchIO BlastIO blast xml parser parse preamble
ExPASy scan seq mirror output
PDB vector mul other
core eval feature fn fn xs classes
core calculate lr x
Graphics GenomeDiagram graph data set data data
core indexed seq file dict repr
Align summary info information content start end e_freq_table log_base
PDB pdblist download entire pdb listfile file_format
SCOP astral is domain in ev dom id
core mutable seq eq other
Pathway Rep graph init nodes
PDB pdblist download obsolete entries listfile file_format
Align summary info dumb consensus threshold ambiguous consensus_alpha require_multiple
Blast xmlparser end element name
PDB rotmat p q
NeuralNetwork BackPropagation input layer backpropagate outputs learning_rate momentum
PopGen GenePop file record skip header
Graphics GenomeDiagram track add set set
Pathway Rep multi graph parent edges child
Pathway network sink interactions species
Nexus tree count terminals node
Graphics organism draw output_file title
SearchIO BlastIO blast xml generator end parent
PDB ppbuilder init radius
SearchIO BlastIO blast xml generator start parents
PDB atom get bfactor
SeqUtils key error neighbors strict
Phylo PAML paml set rel paths
AlignIO phylip iterator split id line
Cluster record save jobname geneclusters expclusters
Restriction defined drop cls
core seq feature bool
Restriction analysis with sites dct
core seq hash
PDB mmtf mmtfparser get structure file_path
SearchIO BlastIO blast xml parser parse hsp root_hsp_frag_elem query_id hit_id
Nexus chain init
NeuralNetwork Gene signature finder add sig sig_dict sig_to_add
Graphics GenomeDiagram feature set str
Align multiple seq alignment getitem index
Graphics GenomeDiagram graph set new graph data name style color
GA Crossover two point crossover init crossover_prob
core feature location len
Entrez data handler element decl name model
PopGen SimCoal sim coal cache init data_dir simcoal_dir
Phylo writer write handle
PDB pdbparser get header
Phylo get child as parent tag construct
phenotype well record setitem time signal
Nexus tree prune taxon
PopGen GenePop gene pop controller test pop hz excess fname enum_test dememorization batches
SeqIO parse handle format alphabet
GenBank split compound loc compound_loc
Graphics GenomeDiagram feature set getitem key
Phylo get children as parent tag construct
Restriction ov3 catalyse cls dna linear
core indexed seq file dict popitem
core mutable seq ne other
HMM markov model builder set random initial probabilities
core bgzf reader readline
Phylo part clades
core compound location init parts operator
Restriction print format make number ls title nc s1
Graphics chromosome component draw
SeqUtils gc123 seq
HMM abstract dpalgorithms backward algorithm
KEGG KGML kgmlparser parse
Nexus nexus utree options
GenBank insdc scanner parse feature feature_key lines
SeqIO sff read raw record handle number_of_flows_per_read
PDB atom get id
PopGen GenePop file record skip population
Phylo phyloxml getitem index
Phylo confidence hash
Pathway Rep graph remove node node
PDB three to one s
core trim str string max_len concat_char
PDB polypeptide get tau list
motifs position weight matrix log odds background
GA Crossover general point crossover do crossover org_1 org_2
Nexus tree get terminals
PDB disordered residue sort
PopGen GenePop easy controller get multilocus f stats
PDB entity delitem id
NeuralNetwork validation increase stop stopping criteria num_iterations training_error validation_error
HMM hidden markov model get blank emissions
AlignIO convert in_file in_format out_file out_format
Restriction analysis change
NeuralNetwork Gene schema factory from motifs motif_repository motif_percent num_ambiguous
core before position position
Phylo phylogeny from clade cls clade
motifs frequency position matrix normalize pseudocounts
Align multiple seq alignment repr
motifs jaspar jaspar5 str
core seq contains char
Blast blast parser end blast output hits
NeuralNetwork BackPropagation hidden layer update previous_layer
core exact position repr
PDB QCPSuperimposer qcpsuperimposer get init rms
HMM abstract trainer estimate params transition_counts emission_counts
Phylo PAML baseml run ctl_file verbose command parse
Restriction restriction type le cls other
core undo handle read size
KEGG KGML reaction substrates
Graphics chromosome draw cur_drawing
SearchIO HmmerIO hmmer3text parser parse preamble
Application option str
SVDSuperimposer svdsuperimposer get transformed
GA Crossover uniform crossover init crossover_prob uniform_prob
Application abstract commandline validate
Graphics get color dict l
Restriction restriction batch search dna linear
SearchIO ExonerateIO set frame frag
Restriction analysis print that dct title s1
Nexus tree init tree weight rooted name
motifs position specific scoring matrix distribution background precision
GenBank feature consumer reference bases content
Blast test
SeqIO seq xml iterator attr seq xml attr_dict record
PDB structure builder set line counter line_counter
PDB vector neg
Graphics GenomeDiagram circular drawer draw sigil cut corner box bottom center top startangle
SearchIO _model query result fragments
Nexus seqmatrix2strmatrix matrix
SVDSuperimposer svdsuperimposer get rms
Sequencing parse handle
codonalign get shift anchor re sh_anc sh_nuc shift_val aa2re
SearchIO BlastIO blast xml indexer get raw offset
Phylo clade root
GenBank insdc scanner parse header
Graphics GenomeDiagram diagram set all tracks attr value
HMM known state trainer train training_seqs
Align print info content summary_info fout rep_record
SeqUtils tm gc seq check strict valueset
PDB disordered entity wrapper sub other
PDB atom flag disorder
codonalign likelihood func t k w pi
codonalign codon alignment getitem index alphabet
HMM markov model builder set random transition probabilities
codonalign get q pi k w codon_lst
HMM markov model builder set random emission probabilities
PDB vector norm
Crystal chain ne other
SearchIO BlastIO blast tab indexer get raw offset
Nexus tree is preterminal node
Alphabet has stop codon contains other
ExPASy parse handle
phenotype plate record update
core exact position position
SearchIO HmmerIO hmmer3tab writer write file qresults
core read forward handle
SearchIO ExonerateIO get scodon moves tmp_seq_blocks
Phylo parser parse values_are_confidence comments_are_confidence rooted
core format alignment align1 align2 score begin
PDB QCPSuperimposer qcpsuperimposer rms coords1 coords2
Phylo distance matrix set zero diagonal
Phylo tree to bitstrs tree
SearchIO _model query result hsp filter func
Pathway reaction eq r
core unknown seq reverse complement
SeqIO sff read seq record handle number_of_flows_per_read flow_chars key_sequence
Nexus tree walk node
Graphics GenomeDiagram circular drawer draw arc arrow inner_radius outer_radius startangle endangle
Phylo writer binary characters obj
Restriction restriction type add cls other
Restriction print format make map only ls title nc s1
motifs jaspar jaspar5 fetch motifs by name name
GenBank loc loc_str expected_seq_length strand
Nexus chain collapse id
Application abstract commandline repr
NMR read dicts fn_list keyatom
Nexus nexus name n vector opts separator
Nexus nexus write nexus data partitions matrix filename blocksize interleave
SearchIO reorient starts starts blksizes seqlen strand
Emboss primers len
core compound location add other
Blast blast parser end iteration query len
SearchIO blat psl indexer iter
SeqUtils check seq method
motifs read handle
NeuralNetwork Gene schema coder init schemas ambiguous_converter
PDB vector get array
PopGen FDist fdist controller generate intfile data_dir
FSSP name filter sum_dict align_dict name_list
core mutable seq reverse complement
codonalign get subgraph codons G
Alphabet alphabet upper
PDB mmtf mmtfparser get structure from url pdb_id
Phylo tree mixin find any
GenBank record consumer start feature table
PDB uniqueify items
core seq lstrip chars
Phylo distance calculator init model
GA generation evolver evolve stopping_criteria
Graphics GenomeDiagram circular drawer draw arc line path start_radius end_radius start_angle
Phylo tree mixin collapse target
Pathway reaction ne r
HMM markov model builder set transition pseudocount from_state to_state count
Restriction ov5 is blunt cls
core calculate nb observation scale
KEGG KGML reaction str
Phylo branch color from hex cls hexstr
SearchIO ExonerateIO get fragments coord frags
SeqIO to dict sequences key_function
SCOP seq map get atoms pdb_handle out_handle
Application abstract commandline str
PDB pdblist get status list url
Entrez elink
core seq tostring
Blast parse handle debug
Phylo accession str
core undo handle iter
PDB mmtf structure decoder set group bond atom_index_one atom_index_two bond_order
GenBank record origin line
Restriction analysis with name names dct
PDB mmtf structure decoder set model info model_id chain_count
Pathway system add reaction reaction
codonalign get pi seq1 seq2 cmethod codon_table
Restriction analysis do not cut start end dct
SearchIO _model hit filter func
Phylo replace wspace text
GenBank record consumer contig location content
Graphics GenomeDiagram feature getattr name
Phylo tree root at midpoint
GA random population genome_alphabet genome_size num_organisms fitness_calculator
core calc p class given x xs classes features alphas
Phylo tree format format
Blast blast parser end hsp qseq
Blast blast parser end hsp align len
core calculate knn x weight_fn distance_fn
PDB read fragments size length dir
core undo handle next
PDB atom get coord
HMM markov model builder all pseudo first_alphabet second_alphabet
HMM markov model builder allow all transitions
codonalign count diff ng86 codon1 codon2 codon_table
Entrez data handler read handle
Phylo PAML parse clademodelc branch_type_no line_floats site_classes
KEGG KGML relation str
Align summary info guess consensus alphabet ambiguous
SeqIO intelli genetics random access get raw offset
NeuralNetwork Gene schema factory init ambiguity_symbol
SearchIO HmmerIO hmmer3text parser parse hit qid qdesc
motifs jaspar record to dict
Alphabet alphabet case less
GenBank record base count line
Graphics comparative scatter plot draw title cur_drawing title width height
Phylo clade iter
KEGG KGML reaction reactant ids
Graphics GenomeDiagram feature get feature
Align multiple seq alignment len
SeqIO gen bank writer write multi line tag text
SCOP node to des record
core indexed seq file dict str
SCOP normalize letters one_letter_code
Nexus node get id
Phylo matrix getitem item
TogoWS open url post
SeqIO imgt iterator handle
SeqIO seq xml writer write seq record
motifs jaspar motif base id
Pathway Rep graph edges label
Restriction restriction batch elements
PDB calc dihedral v1 v2 v3 v4
PDB hsexposure ca pcb vectors pymol filename
PDB disordered entity wrapper getitem id
core indexed seq file dict get k d
SCOP seq map append res
Nexus make unique l
Phylo phylogeny to phyloxml container
core seq feature iter
UniProt gafiterator handle
NeuralNetwork Gene motif finder add motif motif_dict motif_to_add
Phylo PAML ln gamma function alpha
SeqUtils seguid seq
core one of position repr
core indexed seq file proxy get raw offset
Restriction restriction type rshift cls other
Graphics GenomeDiagram circular drawer get feature sigil feature locstart locend
SCOP seq map extend other
Restriction abstract cut neoschizomers cls batch
core indexed seq file dict len
Restriction print format make format cut title nc s1
core feature location nofuzzy end
Phylo parser parse values_are_confidence rooted
Graphics GenomeDiagram diagram add track track track_level
HMM scaled dpalgorithms calculate s value seq_pos previous_vars
motifs jaspar motif init matrix_id name alphabet instances
Align multiple seq alignment format format_spec
PopGen Async async run program id program parameters input_files
codonalign count site ng86 codon_lst k codon_table
PDB abstract hsexposure init model radius offset hse_up_key
NeuralNetwork Gene random motif generator init alphabet min_size max_size
ExPASy read handle
Nexus tree split parent_id n branchlength
SeqIO sff read roche index handle
Application abstract commandline setattr name value
phenotype parse handle format
Phylo bit string independent other
core open filename mode
PDB vector sub other
Restriction not available supplier list cls
NeuralNetwork Gene pattern io read input_handle
codonalign codon seq get full rf table
SeqIO sequence iterator iter
SeqUtils salt correction Na K Tris Mg
PDB ss to index ss
Phylo parser get node info graph context
SearchIO _model hit init hsps id query_id
SCOP scop get ascendent from sql node type
core indexed seq file dict getitem key
core seq upper
PDB select accept residue residue
PDB residue sort a1 a2
SearchIO HmmerIO hmmer2text parser parse preamble
GA Mutation single position mutation init mutation_rate
SeqIO index filename format alphabet key_function
core classify nb observation
motifs jaspar motif version
KEGG Compound parse handle
Nexus chain kill id
NeuralNetwork Gene genetic algorithm finder set up genetic algorithm
NeuralNetwork Gene random motif generator random motif
Phylo object matcher obj
Graphics GenomeDiagram track range
SearchIO ExonerateIO exonerate vulgar indexer get raw offset
codonalign get kappa t pi TV t
PDB QCPSuperimposer qcpsuperimposer set reference_coords coords
SeqIO make seqfeature name from_res to_res description
core seq ungap gap
SearchIO _model query result hit map func
Wise all pairs singles
Nexus tree is internal node
PopGen FDist fdist controller init fdist_dir ext
GA organism ge other
Phylo tree clade
core train iis xs classes features f_sharp
KEGG KGML pathway bounds
Graphics GenomeDiagram graph set getitem key
Application abstract commandline set parameter name value
core train bw states alphabet training_data pseudo_initial
core bgzf reader seek virtual_offset
PopGen SimCoal get demography template stream model tp_dir
PopGen FDist fdist controller get temp file
GenBank base gen bank consumer split accessions accession_string
SearchIO ExonerateIO get inter coords coords strand
SearchIO to dict qresults key_function
Blast blast parser end parameters expect
codonalign yn00 seq1 seq2 k codon_table
Compass record query coverage
Phylo tree mixin is monophyletic terminals
SeqIO ig iterator handle alphabet
SCOP scop write des handle
NeuralNetwork Gene genetic algorithm finder init alphabet
Sequencing read handle
Restriction ambiguous mod2 cls other
Restriction no cut characteristic cls
core seq le other
core seq find sub start end
GA Selection diversity selection select population
Graphics chromosome component init
PopGen GenePop gene pop controller test global hz excess fname enum_test dememorization batches
PDB entity get iterator
TogoWS search iter db query limit batch
SearchIO ExonerateIO exonerate text indexer get raw offset
Pathway Rep graph nodes
NeuralNetwork Gene signature finder find seq_records signature_size max_gap
core bgzf writer flush
SearchIO BlastIO blast tab indexer get raw qresult commented offset
Phylo PAML parse others lines results sequences
Phylo PAML parse distances lines results
core as handle handleish mode
ExPASy sprot search de text swissprot trembl cgi
Alphabet has stop codon upper
Phylo PAML codeml set rel paths
Graphics GenomeDiagram linear drawer canvas location base
Phylo parser read
GA organism le other
PDB select accept model model
core getattr str obj attr fmt fallback
SCOP parse domain str
Restriction blunt catalyse cls dna linear
Graphics kgmlcanvas add orthologs
Graphics GenomeDiagram diagram range
NeuralNetwork BackPropagation output layer init num_nodes activation
SearchIO ExonerateIO flip codons codon_seq target_seq
core find test dir start_dir
SearchIO BlastIO blast xml writer adjust output hsp elem attr
GenBank iterator init handle parser
Application test
core bgzf writer seekable
SearchIO HmmerIO hmmer3domtab parser parse qresult
Graphics chromosome counts add count segment_name count
Alphabet consensus base alphabet alphabets
PDB disordered entity wrapper len
PDB abstract property map iter
Graphics kgmlcanvas add genes
Phylo tree constructor build tree msa
PDB entity reset full id
Phylo parser new clade parent
AlignIO alignment iterator next
SearchIO HmmerIO hmmer2text parser parse qresult
core print matrix matrix
Phylo parser parse tree text
Blast blast parser end iteration query def
PDB polypeptide get sequence
NeuralNetwork Gene pattern repository get top num_patterns
core unknown seq lower
GenBank record version line
SCOP scop get domains
PDB m2rotaxis m
PDB get unique parents entity_list
codonalign igam a x
SeqIO fastq convert tab in_handle out_handle alphabet
core seq feature str
GA Crossover uniform crossover do crossover org_1 org_2
Graphics GenomeDiagram track get ids
PDB atom detach parent
core find start score_matrix align_globally
SeqIO sequence writer get seq string record
Graphics GenomeDiagram abstract drawer len
Nexus compact4nexus orig_list
PDB pdbio init use_model_flag
KEGG Compound record init
Nexus nexus check taxlabels taxon
SearchIO BlastIO blast tab parser get frag frame frag seq_type parsedict
GenBank record parser init debug_level
phenotype well record get signals
Restriction analysis full linear
Phylo PAML baseml read ctl file ctl_file
Restriction print format next section ls into
Graphics kgmlcanvas draw filename
PopGen GenePop file record str
HMM gen random array n
SCOP node to hie record
Graphics GenomeDiagram linear drawer draw sigil octo bottom center top x1
Pathway network source interactions species
motifs motif degenerate consensus
SearchIO _model hsp init fragments
PDB atom get vector
PopGen FDist convert genepop to fdist gp_rec report_pops
motifs jaspar split jaspar id id
PDB vector right multiply matrix
SeqIO ace iterator handle
core bgzf writer tell
PDB unfold entities entity_list target_level
Pathway Rep graph add edge source to label
Nexus tree is identical tree2
SeqUtils protein analysis get amino acids percent
phenotype guess lag x y
core feature location extract parent_sequence
SubsMat build exp freq mat exp_freq_table
Restriction one cut cut once cls
GenBank record accession line
SeqIO insdc writer get annotation str record key default just_first
SearchIO _model hit index hsp
HMM markov model builder all blank first_alphabet second_alphabet
Graphics GenomeDiagram circular drawer draw graph set set
Blast blast error parser parse handle
NeuralNetwork Gene schema all unambiguous
HMM baum welch trainer update emissions emission_counts training_seq forward_vars backward_vars
PopGen Async async get result id
KEGG KGML parse handle debug
SearchIO BlastIO blast xml generator end element name
ExPASy get prosite entry id cgi
NeuralNetwork BackPropagation basic network init input_layer hidden_layer output_layer
Nexus nexus gaponly include_missing
codonalign build pro_align nucl_seqs corr_dict gap_char
core sqlite many seq files dict len
Graphics chromosome segment draw subcomponents cur_drawing
core seq rsplit sep maxsplit
Blast blast parser end statistics hsp len
PopGen GenePop easy controller get genotype count pop_pos locus_name
Graphics GenomeDiagram circular drawer draw track track
PDB entity contains id
codonalign diff codon codon1 codon2 fold_dict
SearchIO blat psl writer build row qresult
PDB refmat p q
Restriction analysis init restrictionbatch sequence linear
Graphics chromosome counts scale segment value segment_name scale_value
Restriction analysis overhang3 dct
core undo handle init handle
PDB fragment mapper getitem res
GenBank record sequence line
Pathway reaction str
codonalign codon seq full translate codon_table stop_symbol
SeqIO get phred quality record
PDB entity id value
PopGen SimCoal sim coal controller init simcoal_dir
SubsMat build log odds mat subs_mat logbase factor round_digit
core sqlite many seq files dict build index
core bgzf reader tell
Phylo parser process clade clade
GenBank parse handle
PDB entity get list
PDB abstract property map getitem key
motifs jaspar read jaspar handle
SearchIO BlastIO blast xml writer write param qresult
core seq lt other
KEGG kegg list database org
Graphics bar chart distribution determine position start_x start_y end_x end_y
Nexus tree has support node
TogoWS get fields url
TogoWS convert data in_format out_format
SearchIO BlastIO blast tab parser parse commented qresult
Align multiple seq alignment iter
Pathway reaction species
Wise parse iterable
Graphics GenomeDiagram diagram move track from_level to_level
GenBank insdc scanner feed first line consumer line
Graphics GenomeDiagram color translator translate color colour
HMM pretty print prediction emissions real_state predicted_state emission_title
Phylo phylogeny get confidence
SCOP seq map index get seq map residues
codonalign codon alignment str
PDB superimposer apply atom_list
motifs jaspar read pfm handle
SeqIO handle convert in_handle in_format out_handle out_format
GenBank feature consumer record end content
Phylo draw tree label_func do_show show_confidence
Graphics GenomeDiagram feature set del feature feature_id
Blast blast parser end hit accession
SearchIO list from csv csv_string caster
SeqIO gen bank writer write the first line record
SearchIO HmmerIO hmmer3text parser parse aln block hid hsp_list
Graphics GenomeDiagram graph data len
PopGen FDist convert genepop to fdist gp_rec
core bgzf reader exit type value traceback
PDB annotate m ss_seq
codonalign ng86 seq1 seq2 k codon_table
SCOP astral domains clustered by ev id
Graphics GenomeDiagram color translator get colorscheme
PDB pdbparser get trailer
Phylo parse handle
Blast scanner feed handle consumer
NeuralNetwork Gene motif coder representation sequence
core seq lower
GA Selection diversity selection get new organism new_pop old_pop
Pathway system remove reaction reaction
SearchIO ExonerateIO get blocks rows coords idx
KEGG KGML pathway remove reaction reaction
Restriction formatted seq init seq linear
SearchIO HmmerIO hmmer3tab writer build header first_qresult
Graphics GenomeDiagram linear drawer draw tick tickpos ctr ticklen track
Phylo matrix delitem item
core transcribe dna
Pathway Rep multi graph init nodes
core bgzf reader fileno
Phylo bootstrap msa times
motifs motif weblogo fname format version
Graphics GenomeDiagram graph data range
PopGen GenePop record remove population pos
codonalign ml seq1 seq2 cmethod codon_table
motifs motif consensus
Graphics GenomeDiagram graph data mean
Restriction restriction type div cls other
NeuralNetwork Gene schema finder find seq_records
core unknown seq transcribe
Graphics GenomeDiagram intermediate points start end graph_data
GenBank record keywords line
Phylo parsimony scorer get score tree alignment
Pathway system species
Phylo indent elem level
motifs score distribution threshold fpr fpr
Blast xmlparser start element name attr
PDB atom sub other
Phylo parser new clade node
Restriction defined is unknown cls
Graphics GenomeDiagram linear drawer draw bar graph graph
KDTree neighbor test nr_points dim bucket_size radius
Blast blast parser end hsp midline
core indexed seq file dict get raw key
GenBank insdc scanner feed header lines consumer lines
SearchIO BlastIO blast tab indexer iter
Phylo matches s
Pathway network init species
Graphics GenomeDiagram graph set len
Graphics GenomeDiagram graph set get graphs
PDB entity get full id
Restriction restriction type ge cls other
core compound location start
Restriction abstract cut is equischizomer cls other
Phylo combine matchers target kwargs require_spec
Graphics GenomeDiagram linear drawer draw feature feature
Alphabet check type compatible alphabets
GenBank feature consumer molecule type mol_type
Nexus tree merge with support bstrees constree threshold outgroup
Align multiple seq alignment append record
core bgzf reader close
SCOP seq map process line
SeqUtils codon adaptation index cai for gene dna_sequence
Graphics GenomeDiagram color translator get artemis colorscheme
Align multiple seq alignment append record expected_length
core seq record str
PDB abstract property map has key id
core find gap open sequenceA sequenceB ali_seqA ali_seqB
PDB entity add entity
Graphics GenomeDiagram track str
PDB run psea fname
SearchIO HmmerIO base hmmer text indexer get raw offset
Phylo PAML parse branch site a foreground line_floats site_classes
SeqIO fastq solexa iterator handle alphabet title2ids
SCOP parse handle
KEGG KGML read handle debug
SeqIO fasta writer init handle wrap record2title
SCOP astral get astral domains from sql column
SearchIO BlastIO augment blast hsp hsp attr
PDB pdbparser handle pdb exception message line_counter
Alphabet gapped contains other
Graphics GenomeDiagram linear drawer draw graph set set
UniProt gpi10iterator handle
Graphics write drawing output_file format dpi
PDB mmtf structure decoder init structure total_num_bonds total_num_atoms total_num_groups total_num_chains
core indexed seq file dict contains key
AlignIO maf iterator handle seq_count alphabet
Phylo clade from clade cls clade
Phylo writer info factory plain confidence_as_branch_length branch_length_only max_confidence
Graphics get temp imagefilename url
Graphics GenomeDiagram linear drawer draw sigil jaggy bottom center top x1
Nexus nexus set original taxon order value
Pathway reaction hash
phenotype well record len
Nexus chain unlink id
GenBank insdc scanner feed feature table consumer feature_tuples
core seq init data alphabet
SeqIO paired fasta qual iterator fasta_handle qual_handle alphabet title2ids
codonalign count replacement codon_set G
phenotype plate record subtract control control wells
HMM baum welch trainer train training_seqs stopping_criteria dp_method
PDB structure builder set anisou anisou_array
SearchIO BlastIO blast tab writer build comments qres
NeuralNetwork Gene pattern repository init pattern_info
SeqIO insdc writer split contig record max_len
codonalign dijkstra graph start end
SeqIO index db index_filename filenames format alphabet
core undo handle enter
Restriction commercially available supplier list cls
Graphics chromosome counts get segment info
SubsMat seq mat str
Nexus kill comments and break lines text
GenBank iterator next
SearchIO BlastIO blast xml writer write elem block block_name map_name obj opt_dict
HMM markov model builder set transition score from_state to_state probability
SeqUtils crc64 s
Graphics GenomeDiagram feature set len
Pathway Rep multi graph children parent
Phylo parser parse
KEGG KGML entry remove component value
SeqIO sequential sequence writer write record record
Phylo postorder traverse root get_children
Nexus tree set subtree node
SearchIO ExonerateIO base exonerate parser read until bool_func
SearchIO ExonerateIO comp coords hsp seq_type inter_lens
Nexus node set prev id
GA Selection roulette wheel selection set up wheel population
core translate str sequence table stop_symbol to_stop
Restriction print format format output dct title s1
TogoWS search count db query
SeqUtils lcc simp seq
GenBank record locus line
Nexus tree common ancestor node1 node2
PDB psea get seq
core classify lr x
Nexus nexus write nexus data filename matrix exclude delete
Restriction ambiguous elucidate cls
PopGen GenePop gene pop controller calc ibd haplo fname stat scale min_dist
SearchIO HmmerIO hmmer3text parser read until bool_func
AlignIO sequential alignment writer write footer
Restriction restriction batch lambdasplit func
core sqlite many seq files dict init index_filename filenames proxy_factory format
PopGen GenePop gene pop controller test global hz both fname type ext enum_test
Graphics GenomeDiagram graph set str
motifs position specific scoring matrix calculate sequence
core compound location contains value
Phylo sorted attrs elem
core bgzf writer exit type value traceback
PDB hsexposure cb get cb r1 r2 r3
Phylo parser parse clade parent
HMM hidden markov model init initial_prob transition_prob emission_prob transition_pseudo
Nexus nexus bootstrap matrix delete exclude
codonalign check corr pro nucl gap_char codon_table
Graphics GenomeDiagram diagram renumber tracks low step
SeqUtils crc32 seq
Blast xmlparser characters ch
core bgzf writer write data
PDB mmtf structure decoder set inter group bond atom_index_one atom_index_two bond_order
AlignIO sequential alignment writer write file alignments
Restriction non palindromic search cls
SCOP astral write to sql db_handle
Pathway Rep multi graph parents child
Phylo tree mixin common ancestor targets
core is blank line line allow_spaces
HMM abstract trainer ml estimator counts
SCOP node get children
AlignIO stockholm iterator get meta data identifier meta_dict
GenBank feature consumer accession acc_num
core load bgzf block handle text_mode
core indexed seq file dict copy
PDB mmtf structure decoder set xtal info space_group unit_cell
Restriction not available is comm cls
Restriction palindromic search cls
SeqIO fastq solexa convert fastq sanger in_handle out_handle alphabet
NeuralNetwork Gene schema find ambiguous motif
core unknown position position
Pathway Rep graph parent edges child
Blast blast parser end hsp positive
Graphics GenomeDiagram cross link init featureA featureB color border
Phylo PAML codeml run ctl_file verbose command parse
Graphics GenomeDiagram abstract drawer set page size pagesize orientation
Blast blast parser end hsp score
core compound location str
core seq record iter
PDB disordered entity wrapper getattr method
Restriction analysis defined dct
SearchIO _model hit id all
KEGG KGML pathway maps
Phylo clade bool
SeqIO pdb atom iterator handle
motifs generic position matrix consensus
SeqIO tab writer write record record
SeqUtils make table oldtable values
AlignIO maf index init sqlite_file maf_file target_seqname
core seq len
core make score matrix fast sequenceA sequenceB match_fn open_A
Nexus nexus invert charlist
PDB entity get id
Phylo nnitree searcher search starting_tree alignment
Emboss input record add primer set primer_name first_primer_seq second_primer_seq
Pathway reaction reverse
Cluster record distancematrix transpose dist
SeqIO parse tag data elem_code elem_num raw_data
Phylo confidence floordiv other
Pathway Rep multi graph str
Restriction not defined elucidate cls
Phylo preorder traverse root get_children
PDB dssp dict from pdb file in_file DSSP
Blast blast error parser diagnose error handle data_record
Graphics GenomeDiagram abstract drawer set margins x y xl xr
Data register ncbi table name alt_name id table
core seq get seq str and check alphabet other_sequence
SeqIO fastq solexa convert fastq illumina in_handle out_handle alphabet
SeqIO solexa quality from phred phred_quality
SeqIO gen bank writer write record record
GenBank record comment line
Phylo adam consensus trees
Nexus nexus export fasta filename width
Alphabet alphabet encoder upper
KEGG KGML pathway compounds
core feature location parts
KEGG kegg find database query option
Sequencing parse handle
PopGen FDist fdist async run job parameters input_files
core within position extension
HMM baum welch trainer update transitions transition_counts training_seq forward_vars backward_vars
AlignIO identifier split identifier
Nexus node get data
Phylo phyloxml len
SeqIO sff random access iter
KDTree kdtree search center radius
GenBank feature consumer dblink content
Nexus nexus read input
NMR find start entry line n
PopGen GenePop read fname
KEGG KGML pathway remove relation relation
core undo handle readlines
core seq transcribe
HMM markov model builder set emission pseudocount seq_state emission_state count
GenBank feature consumer project content
Phylo branch color from name cls colorname
SeqIO embl cds feature iterator handle alphabet
Graphics annotated chromosome segment init bp_length features default_feature_color name_qualifiers
Graphics GenomeDiagram linear drawer draw greytrack track
NeuralNetwork Gene schema factory schema from motif motif motif_list num_ambiguous
Restriction analysis overhang5 dct
Pathway Rep graph eq g
Graphics GenomeDiagram linear drawer draw track track
PDB pdbio save file select write_end preserve_atom_numbering
SearchIO BlastIO blast xml parser parse hit root_hit_elem query_id
NeuralNetwork Gene pattern io init alphabet
phenotype logistic x A u d
SCOP search pdb key sid disp
Graphics GenomeDiagram feature set range
GenBank record source line
core seq record format format_spec
PDB chain has id id
PDB pdbparser parse coordinates coords_trailer
UniProt gpa11iterator handle
Pathway Rep graph children parent
Nexus nexus parse nexus block title contents
Graphics GenomeDiagram linear drawer draw sigil big arrow bottom center top x1
motifs jaspar write motifs format
codonalign get aa regex codon_table stop unknown
Phylo sequence from seqrecord cls record is_aligned
HMM scaled dpalgorithms forward recursion cur_state sequence_pos forward_vars
ExPASy read handle
PopGen GenePop gene pop controller calc ibd fname sub stat scale
GenBank feature consumer taxonomy content
Restriction ambiguous is ambiguous cls
Graphics GenomeDiagram linear drawer set track heights
core bgzf blocks handle
Blast xmlparser secure name name
Graphics GenomeDiagram graph data stdev
PDB one to index s
PDB atom get full id
SearchIO _model base search object transfer attrs obj
Phylo phyloxml iter
GenBank record consumer sequence content
SearchIO HmmerIO hmmer3tab indexer get raw offset
Statistics lowess x y f iter
Graphics GenomeDiagram circular drawer draw arc poly inner_radius outer_radius inner_startangle inner_endangle
Restriction restriction type str cls
core one of position extension
PDB fragment add residue resname ca_coord
GA Selection roulette wheel selection init mutator crossover repairer
Nexus nexus parse list options_buffer set_type
SeqIO embl convert fasta in_handle out_handle alphabet
motifs read handle format
Nexus adjust lines lines
core make score matrix generic sequenceA sequenceB match_fn gap_A_fn
SubsMat substitution matrix calculate relative entropy obs_freq_mat
core feature location add other
NeuralNetwork Gene schema num ambiguous motif
AlignIO maf index get record offset
SeqIO simple fasta parser handle
Phylo to networkx tree
PDB atom get anisou
Restriction analysis only between start end dct
GA Mutation conversion mutation init mutation_rate
Restriction unknown mod1 cls other
PDB entity iter
Pathway system stochiometry
PopGen GenePop gene pop controller calc allele genotype freqs fname
KEGG KGML pathway remove entry entry
Nexus tree is monophyletic taxon_list
Nexus combine matrices
PDB QCPSuperimposer qcpsuperimposer run
core mutable seq complement
core exact position extension
SeqIO sequence writer clean text
Graphics distribution page draw title cur_drawing title width height
Nexus nexus get original taxon order
Phylo resolve uri s namespaces cdao_to_obo xml_style
AlignIO maf index get spliced starts ends strand
SeqIO get read time read_name
GenBank feature value cleaner clean value key_name value
Restriction blunt is 5overhang cls
Restriction analysis with site size site_size dct
Pathway Rep multi graph remove node node
Phylo tree is terminal
SeqIO seq xml iterator attr dbref attr_dict record
PopGen GenePop easy controller get alleles all pops locus_name
Nexus tree collapse genera space_equals_underscore
core seq startswith prefix start end
Phylo clean attrib obj attrs
PopGen GenePop gp int tok
Graphics kgmlcanvas add reaction entries
SeqIO fastq solexa convert qual in_handle out_handle alphabet
phenotype well record add well
Pathway reaction repr
core indexed seq file dict clear
SearchIO BlastIO blast tab writer adjust output field value
PDB disordered entity wrapper disordered get list
Nexus tree distance node1 node2
Restriction analysis sub set wanted
core after position position
Application switch str
Blast blast parser end statistics db num
Phylo sequence to seqrecord
PDB fast mmcifparser init structure_builder QUIET
PopGen GenePop easy controller estimate nm
SearchIO BlastIO blast xml generator start parent name attrs
SearchIO HmmerIO hmmer2text parser push back line
motifs position specific scoring matrix search sequence threshold both
phenotype well record sub well
Wise build align cmdline cmdline pair output_filename kbyte
SVDSuperimposer svdsuperimposer set reference_coords coords
core indexed seq file dict fromkeys keys value
KEGG KGML entry bounds
PDB vector copy
SubsMat seq mat sub other
GenBank base gen bank consumer clean location location_string
Graphics GenomeDiagram diagram get drawn levels
core train xs ys update_fn typecode
PDB fragment mapper map model
Pathway interaction hash
Phylo branch color to rgb
Phylo tree mixin find elements target terminal order
Phylo local tag
Nexus nexus codonposset options
Sequencing acefile record sort
PDB fragment get resname list
core back transcribe rna
Restriction restriction batch rtruediv other
Phylo parser parse phylogeny parent
SeqIO sequential seq file random access iter
Restriction not defined is ambiguous cls
AlignIO alignment writer write file alignments
TogoWS search db query offset limit
Phylo distance calculator get distance msa
SeqIO fastq illumina iterator handle alphabet title2ids
Phylo strict consensus trees
SVDSuperimposer svdsuperimposer get init rms
Nexus standard data str
SeqIO tab iterator handle alphabet
PopGen SimCoal fast sim coal controller init fastsimcoal_dir bin_name
GA Mutation conversion mutation mutate organism
PDB neighbor search search all radius level
Entrez parse handle validate
AlignIO emboss writer write alignment alignment
Nexus nexus append sets exclude delete mrbayes include_codons
SearchIO get aln slice coords parsed_hsp
Phylo PAML paml get all options
SeqUtils protein analysis isoelectric point
Nexus tree sum branchlength root node
Phylo PAML parse parameters lines results num_params
SearchIO HmmerIO hmmer3text parser parse qresult
Phylo distance calculator pairwise seq1 seq2
Phylo level traverse root get_children
motifs motif reverse complement
KEGG wrap kegg line max_width wrap_rule
Graphics GenomeDiagram linear drawer draw sigil box bottom center top x1
SCOP scop get domain from sql sunid sid
core recover alignments sequenceA sequenceB starts score_matrix
GenBank record consumer primary ref line content
PDB atom is disordered
PopGen GenePop gene pop controller test pop hz both fname type ext enum_test
SearchIO BlastIO blast xml writer write hits hits
core undo handle exit type value traceback
Cluster record clusterdistance index1 index2 method dist
SearchIO _model hit map func
PDB model sort c1 c2
KEGG KGML graphics centre
GenBank record consumer feature qualifier name content_list
core compound location nofuzzy end
PopGen GenePop record remove locus by position pos
Phylo get child text parent tag construct
GenBank insdc scanner feed handle consumer do_features
Graphics GenomeDiagram track getitem key
GA organism copy
core undo handle readline
core compound location repr
KEGG KGML reaction add product product_id
Graphics GenomeDiagram draw box point1 point2 color border
SeqIO seq xml writer write dbxrefs record
Blast blast parser end parameters sc mismatch
SearchIO set qresult hits qresult hit_rows
PopGen GenePop file record seek position pop indiv
Align pssm get residue pos
codonalign mktest codon_alns codon_table alpha
KEGG KGML reaction products
PDB abstract hsexposure get gly cb vector residue
Phylo parser parse children node
Phylo PAML yn00 write ctl file
SearchIO singleitem attr doc
Graphics GenomeDiagram feature set to string verbose
core seq complement
SearchIO ExonerateIO make triplets seq phase
Phylo bootstrap consensus msa times tree_constructor consensus
Graphics chromosome segment overdraw subcomponents cur_drawing
Restriction defined is ambiguous cls
PDB residue get unpacked list
SearchIO ExonerateIO comp split codons hsp seq_type scodon_moves
NMR peaklist write header outfn
Pathway network add species species
PDB fragment get id
Restriction abstract cut frequency cls
NeuralNetwork Gene pattern repository get all
Entrez esearch db term
SubsMat log odds matrix calculate relative entropy obs_freq_mat
Restriction ambiguous is unknown cls
Phylo serialize value
core unknown seq str
PDB mmtf structure decoder set chain info chain_id chain_name num_groups
codonalign q i j pi k
Alphabet consensus alphabet alphabets
SeqIO insdc location string location rec_length
PopGen Async file retriever get file list
Nexus get start end sequence skiplist
core align sequenceA sequenceB match_fn gap_A_fn
Phylo get support target_tree trees len_trees
NeuralNetwork Gene signature coder representation sequence
SearchIO BlastIO blast tab indexer qresult index
Graphics GenomeDiagram circular drawer draw greytrack track
SeqUtils x gc skew seq window zoom r
GA Mutation single position mutation mutate organism
Data list ambiguous codons codons ambiguous_nucleotide_values
Align multiple seq alignment add other
AlignIO relaxed phylip writer write alignment alignment
Graphics GenomeDiagram draw arrow point1 point2 color border
PDB superimposer set atoms fixed moving
Nexus tree add nodedata nd st
GenBank base gen bank consumer remove newlines text
Restriction formatted seq is linear
Phylo nnitree searcher nni starting_tree alignment
Phylo tree mixin get nonterminals order
HMM markov model builder allow transition from_state to_state probability pseudocount
HMM abstract trainer log likelihood probabilities
motifs jaspar jaspar5 fetch motifs collection tf_name tf_class tf_family
SearchIO _model query result hsp map func
Phylo write obj handle
GenBank insdc scanner parse features skip
PDB vector pow other
Blast blast parser end blast output program
KEGG KGML entry element
PDB disordered residue disordered add residue
Restriction restriction batch get enzyme add
SeqUtils protein analysis flexibility
Restriction unknown is 3overhang cls
core unknown seq add other
core read and call until uhandle method
core unknown seq count sub start end
NeuralNetwork Gene signature finder get signature dict seq_records sig_size max_gap
Graphics chromosome init chromosome_name
SearchIO BlastIO blast xml generator simple element name content
core mutable seq toseq
SearchIO HmmerIO hmmer2text parser read next rstrip
Nexus nexus weighted stepmatrix name exclude delete
GA Selection tournament selection init mutator crossover repairer num_competitors
NeuralNetwork Gene most count schema fitness calculate fitness genome
Nexus char buffer rest
core bgzf reader read size
PopGen GenePop record str
Graphics distribution page draw legend cur_drawing start_y width
KEGG write kegg item info indent
Phylo majority consensus trees cutoff
SearchIO ExonerateIO create hsp hid qid hspd
Graphics GenomeDiagram linear drawer draw scale track
codonalign codon alignment from msa cls align alphabet
SearchIO ExonerateIO split fragment frag
Graphics GenomeDiagram graph set range
SeqUtils codon adaptation index print index
FSSP filter sum_dict align_dict filter_attribute low_bound
PDB pdblist get seqres file savefile
PopGen GenePop easy controller get heterozygosity info pop_pos locus_name
Phylo parser parse
Alphabet verify alphabet sequence
Phylo parser make tree cls node node_dict children
SeqIO sequence iterator next
KEGG kegg get dbentries option
Nexus nexus export phylip filename
Graphics color to reportlab color
core complement sequence
Phylo bit string new cls strdata
core train xs ys k typecode
NeuralNetwork BackPropagation input layer update inputs
AlignIO mauve writer write record record record_idx
core unknown position repr
codonalign codon alignment get dn ds matrix method codon_table
GenBank reference medline line
core seq record contains char
core unknown position extension
Phylo write trees handle plain
AlignIO maf index search starts ends
PDB pdbio get atom line atom hetfield segid atom_number
Graphics GenomeDiagram graph data get data
core indexed seq file proxy get offset
SeqUtils six frame translations seq genetic_code
Phylo PAML codeml init alignment tree working_dir out_file
Graphics GenomeDiagram linear drawer init parent pagesize orientation x
core unknown seq translate
Entrez epost db
motifs position specific scoring matrix min
core indexed seq file dict init random_access_proxy key_function repr obj_repr
SeqIO uniprot iterator handle alphabet return_raw_comments
SearchIO ExonerateIO exonerate text parser read alignment
AlignIO maf writer write alignment alignment
HMM baum welch trainer init markov_model
SCOP astral hashed domains by ev id
Graphics bar chart distribution draw cur_drawing start_x start_y end_x
core maketrans complement_mapping
SeqIO phd iterator handle
SearchIO ExonerateIO parse hit or query line line
Align summary info pos specific score matrix axis_seq chars_to_ignore
SearchIO fullcascade attr doc
Restriction restriction type init cls name bases dct
SCOP index getitem key
Pathway Rep graph str
Phylo PAML baseml write ctl file
GA Crossover safe fitness crossover do crossover org_1 org_2
Nexus standard data len
Alphabet alphabet encoder contains other
Phylo confidence truediv other
GenBank feature consumer feature qualifier key value
Graphics GenomeDiagram feature set add feature feature
core within position position
SeqIO swiss iterator handle
Application escape filename filename
core seq strip chars
KEGG KGML pathway genes
PopGen Async async run program program parameters input_files
core compound location end
PDB disordered entity wrapper setitem id child
ExPASy read handle
core seq record reverse complement id name description features
Graphics chromosome draw labels cur_drawing left_labels right_labels
motifs write motifs
SearchIO HmmerIO hmmer3domtab hmmhit writer build header first_qresult
motifs score distribution threshold fnr fnr
NeuralNetwork Gene schema num matches motif query
Graphics GenomeDiagram graph data getitem index
NeuralNetwork Gene schema coder representation sequence
phenotype plate record is well obj
Phylo tree mixin is bifurcating
KEGG Enzyme record str
Phylo parser parse graph graph context
PDB abstract hsexposure get cb r1 r2 r3
Restriction formatted seq circularise
Blast qblast program database sequence url_base
Blast blast parser reset
SeqIO gen bank iterator handle
Pathway Rep multi graph nodes
Graphics chromosome subcomponent size
SeqIO fastq sanger convert fastq illumina in_handle out_handle alphabet
PopGen SimCoal sim coal cache list simulations ploidy
CAPS differential cutsite init
SeqUtils protein analysis secondary structure fraction
HMM calculate to transitions trans_probs
SeqIO fastq illumina convert qual in_handle out_handle alphabet
PDB structure builder init atom name coord b_factor occupancy
Graphics GenomeDiagram linear drawer get feature sigil feature x0 x1 fragment
SeqIO gen bank writer get topology record
PDB map fragment list flist reflist
Phylo tree randomized cls taxa branch_length branch_stdev
motifs motif len
SeqIO seq xml writer write species record
core one of position new cls position choices
PDB structure builder init model model_id serial_num
AlignIO alignment iterator init handle seq_count alphabet
PDB fragment mapper init model lsize flength fdir
PDB structure builder set sigatm sigatm_array
Restriction abstract cut is neoschizomer cls other
Restriction print format init
Restriction unknown is blunt cls
core bgzf reader seekable
GenBank wrapped genbank information indent wrap_space split_char
core seq record getitem index
SearchIO _model query result sort key reverse in_place
GenBank base gen bank consumer split taxonomy taxonomy_string
core unknown seq getitem index
Nexus tree add subtree parent_id tree
Restriction blunt compatible end cls batch
Restriction print format make nocut ls title nc s1
core indexed seq file proxy iter
Align multiple seq alignment format format
core equal weight x y
Phylo date str
core between position position
NeuralNetwork Gene schema factory get unique schema cur_schemas motif_list num_ambiguous
PDB pdblist get all obsolete
core compound location ref
core seq feature init location type location_operator strand
GenBank record consumer add feature
Pathway Rep multi graph labels
SeqIO seq xml iterator attr property attr_dict record
Pathway Rep graph parents child
phenotype guess plateau x y
SearchIO BlastIO blast xml writer write preamble qresult
PopGen FDist convert genepop to fdist big gp_rec report_pops
Restriction ov5 compatible end cls batch
SubsMat seq mat print mat f format bottomformat alphabet
NeuralNetwork Gene motif finder init alphabet_strict
Restriction abstract cut isoschizomers cls batch
GenBank record segment line
GenBank insdc scanner parse footer
Phylo parsimony tree constructor build tree alignment
NeuralNetwork Gene matches schema pattern schema ambiguity_character
NeuralNetwork Gene pattern repository get differing top_num bottom_num
SearchIO BlastIO blast tab writer write file qresults
PDB psea2hec pseq
PDB atom get parent
Graphics chromosome counts init segment_names color_scheme
codonalign merge aa2re aa1 aa2 shift_val aa2re
GenBank feature consumer size content
Restriction analysis blunt dct
PDB extract structure chain_id start end
Phylo write trees file format
PDB disordered entity wrapper iter
motifs jaspar jaspar5 fetch internal id base_id version
Phylo PAML parse basics lines results
Medline parse handle
core train training_set results priors typecode
HMM calculate from transitions trans_probs
Phylo matrix insert name value index
GA Crossover general point crossover crossover x no locs
Phylo PAML codeml read ctl file ctl_file
PDB pdblist print default format warning file_format
PDB entity detach child id
GenBank record features line
core bgzf writer init filename mode fileobj compresslevel
PDB disordered atom disordered add atom
Align summary info get base replacements skip_items
AlignIO sequential alignment writer write alignment alignment
Graphics chromosome draw label cur_drawing label_name
Graphics GenomeDiagram graph set to string verbose
SeqIO embl writer write the first lines record
Phylo matrix setitem item value
AlignIO seq io to alignment iterator handle format alphabet seq_count
Blast blast parser end blast output reference
core indexed seq file dict items
Phylo cdao to obo s
Graphics GenomeDiagram diagram get tracks
SeqIO read handle format alphabet
SCOP scop get domain by sid sid
SearchIO BlastIO blast xml generator start document
HMM scaled dpalgorithms backward recursion cur_state sequence_pos backward_vars
SeqIO sff writer write file records
Blast blast parser end blast output query len
Graphics GenomeDiagram diagram getitem key
Graphics bar chart distribution init display_info
SeqIO qual phred writer write record record
phenotype plate record getitem index
PDB one to three s
Restriction analysis show only between start end dct
Graphics kgmlcanvas draw arrow g_from g_to
core calculate me observation
Restriction restriction batch split
Phylo phylogeny to alignment
SeqIO get sanger quality str record
GenBank feature parser init debug_level use_fuzziness feature_cleaner
phenotype plate record get column column
Phylo writer process clade clade parent root
motifs jaspar jaspar5 fetch counts matrix int_id
GenBank embl scanner parse footer
Phylo PAML parse nssites lines results multi_models multi_genes
core clean alignments alignments
Graphics GenomeDiagram graph set data quartiles
Nexus nexus get nexus block file_contents
Phylo split namespace tag
SearchIO ExonerateIO get block coords parsed_seq row_dict has_ner
Compass parse handle
GA organism gt other
Alphabet alphabet contains other
SeqIO fastq general iterator handle
Restriction meth dep is methylable cls
Restriction one cut characteristic cls
Phylo count clades trees
Restriction blunt mod1 other
core feature location init start end strand ref
Graphics GenomeDiagram diagram del track track_level
Restriction non palindromic is palindromic cls
SCOP scop get node by sunid sunid
Phylo tree mixin count terminals
Phylo PAML yn00 read ctl file ctl_file
Align multiple seq alignment add sequence descriptor sequence start end
Restriction restriction batch suppl codes cls
ExPASy read handle
Restriction restriction type eq cls other
GenBank record pid line
Nexus chain link parent child
phenotype json writer write handle
core sqlite many seq files dict iter
phenotype write plates handle format
ExPASy get prodoc entry id cgi
GenBank record consumer feature key content
GenBank feature value cleaner init to_process
Nexus sort keys by values p
SCOP record process line
codonalign codon seq translate codon_table stop_symbol rf_table ungap_seq
SearchIO parse handle format
Phylo parser parse handle to graph rooted parse_format context
Blast blast parser parse handle
NeuralNetwork BackPropagation basic network predict inputs
KEGG KGML pathway get kgml
AlignIO fasta m10iterator handle alphabet
SearchIO BlastIO blast xml parser parse qresult
Restriction restriction batch as string
GenBank feature consumer contig location content
KEGG KGML pathway reactions
core sqlite many seq files dict close
Phylo attribute matcher kwargs
Phylo tree mixin is preterminal
Graphics GenomeDiagram draw cut corner box point1 point2 corner color
PDB pdblist download pdb files pdb_codes obsolete pdir file_format
Blast blast parser end hsp bit score
SeqUtils chem correction melting_temp DMSO fmd DMSOfactor
core seq rfind sub start end
core mutable seq str
Phylo draw graphviz tree label_func prog args
Nexus nexus adjust charlabels exclude insert
Restriction analysis without site dct
PDB structure alignment test equivalence r1 aa1
Nexus nexus resolve identifier set_type
PDB atom set parent parent
SeqIO seq xml iterator init handle
HMM scaled dpalgorithms init markov_model sequence
PDB atom transform rot tran
NeuralNetwork Gene signature finder init alphabet_strict
GenBank record consumer record end content
motifs jaspar jaspar5 init host name user password
Graphics GenomeDiagram circular drawer init parent pagesize orientation x
core position gap repr
GenBank base gen bank consumer remove spaces text
core seq eq other
SearchIO ExonerateIO comp intron lens seq_type inter_blocks raw_inter_lens
Graphics GenomeDiagram page sizes size
core mutable seq add other
PopGen GenePop file record remove population pos fname
PDB pdblist retrieve pdb file pdb_code obsolete pdir file_format
NeuralNetwork BackPropagation output layer backpropagate outputs learning_rate momentum
GenBank insdc scanner parse records handle do_features
NeuralNetwork Gene motif finder find differences first_records second_records motif_size
KEGG KGML entry add component element
Restriction not defined drop cls
NeuralNetwork example manager init training_percent validation_percent
SeqIO convert in_file in_format out_file out_format
phenotype well record fit function
SearchIO _model query result hsps
PopGen FDist fdist controller run fdist npops nsamples fst sample_size
PDB structure alignment get maps
PDB chain sort r1 r2
PDB pdbparser init PERMISSIVE get_header structure_builder QUIET
Phylo matrix init names matrix
core undo handle tell
Graphics kgmlcanvas add relations
HMM markov model builder set equal probabilities
Restriction unknown overhang cls
SeqIO xmlrecord iterator attributes node
GenBank reference authors line
SeqIO get solexa quality str record
Wise parse line line
Restriction not defined is defined cls
core unknown seq len
SearchIO HmmerIO hmmer2text parser parse hsp alignments
core bgzf reader enter
SeqIO sff find roche index handle
SearchIO allitems attr doc
Phylo parse file
core seq str
GenBank reference title line
GenBank feature consumer feature qualifier description content
Restriction two cuts cut once cls
core compound location nonzero
NMR data table fn_list datalabel keyatom
Blast blast parser end hsp hit frame
Pathway Rep graph labels
core seq reverse complement
SearchIO _model hit append hsp
Graphics GenomeDiagram abstract drawer set bounds start end
PDB structure builder init structure structure_id
Phylo distance calculator build protein matrix subsmat
Entrez einfo
PDB atom get fullname
Align summary info get base letters letters
Restriction analysis format output dct title s1
SeqIO seq xml iterator elem rnaseq node record
Graphics color spiral get colors k offset
Graphics GenomeDiagram circular drawer draw sigil box bottom center top startangle
motifs generic position matrix gc content
Phylo PAML codeml print options
Wise align cmdline pair kbyte force_type
PDB atom init name coord bfactor occupancy
SeqIO sff random access get raw offset
SearchIO BlastIO blast tab parser get frag strand frag seq_type parsedict
PopGen GenePop file record remove loci by name names fname
Restriction defined elucidate cls
PDB polypeptide get ca list
Restriction restriction batch show codes cls
PDB mmtf structure decoder set header info r_free r_work resolution title
SeqIO tab random access get raw offset
core iterlen items
GenBank reference reference line
NeuralNetwork Gene motif coder init motifs
Phylo parse handle
Phylo collapse wspace text
Graphics GenomeDiagram graph data quartiles
Restriction restriction batch remove other
core indexed seq file dict update
PDB ppbuilder test dist c n
Entrez read handle validate
Graphics comparative scatter plot draw scatter plot cur_drawing x_start y_start x_end
AlignIO nexus iterator handle seq_count
core seq feature repr
Phylo PAML read results_file
Restriction restriction type gt cls other
SeqIO sequential sequence writer write records records
SCOP scop write cla handle
SeqIO force alphabet record_iterator alphabet
KEGG KGML reaction entry
Phylo tree mixin is parent of target
motifs score distribution threshold balanced rate_proportion return_rate
phenotype plate record str
UniProt writerec outrec handle fields
Restriction not available buffers cls supplier
HMM hidden markov model log transform probability
SCOP scop init cla_handle des_handle hie_handle dir_path
SeqUtils tm staluc s dnac saltc rna
Restriction ov3 mod1 cls other
PDB chop end codes line
Phylo tree mixin total branch length
Phylo class matcher target_cls
SearchIO blat psl parser parse row
Phylo q uri s
GenBank reference journal line
SearchIO ExonerateIO base exonerate indexer iter
SCOP scop get descendents from sql node type
GenBank base gen bank consumer split keywords keyword_string
Phylo PAML baseml set rel paths
Phylo tree str
core undo handle seek
core open for random access filename
PDB atom get sigatm
Phylo bitstring topology tree
Restriction ov5 is 5overhang cls
PDB atom set sigatm sigatm_array
codonalign get tv codon_lst1 codon_lst2 codon_table
core finish backtrace sequenceA sequenceB ali_seqA ali_seqB
SubsMat build obs freq mat acc_rep_mat
codonalign igamc a x
SearchIO fasta m10parser parse hit table
Pathway network repr
Blast blast parser end hsp hit from
ExPASy parse handle
PopGen FDist fdist controller run fdist force fst npops nsamples fst sample_size
GA Selection abstract selection init mutator crossover repairer
core seq record format format
GA Mutation safe fitness mutation init actual_mutation accept_less
Blast blast parser end hsp evalue
PDB read vertex array filename
KDTree kdtree all get indices
SCOP record process line
KDTree test nr_points dim bucket_size radius
core find string trie
HMM hidden markov model viterbi sequence state_alphabet
Align multiple seq alignment init records alphabet annotations
motifs jaspar jaspar5 is latest version int_id
Blast parse qblast ref page handle
Graphics GenomeDiagram linear drawer draw sigil arrow bottom center top x1
core seq back transcribe
Restriction palindromic is palindromic cls
core compound location extract parent_sequence
GenBank insdc scanner find start
core bgzf reader isatty
Graphics GenomeDiagram color translator int255 color values
Pathway network species
KEGG kegg link target_db source_db option
PopGen GenePop gene pop controller test pop hz deficiency fname enum_test dememorization batches
Phylo PAML parse basics lines results
SeqIO fasta iterator handle alphabet title2ids
KEGG kegg conv target_db source_db option
motifs position specific scoring matrix std background
UniProt writebyproteinrec outprotrec handle fields
Blast blast parser end hsp query from
core calc f sharp N nclasses features
PDB index to one index
core bgzf reader next
PDB mmcifparser get structure structure_id filename
Entrez ecitmatch
core seq rstrip chars
PDB QCPSuperimposer qcpsuperimposer get rms
Restriction no cut rev modify cls location
Phylo writer domain obj
codonalign codon seq get codon num
Graphics GenomeDiagram linear drawer draw heat graph graph
NeuralNetwork BackPropagation abstract layer init num_nodes has_bias_node
Affy read v4 f
Align multiple seq alignment sort key reverse
SeqIO fastq convert fasta in_handle out_handle alphabet
Graphics GenomeDiagram feature set colour colour
Phylo PAML read results_file
Restriction restriction batch div other
Nexus node set id id
Pathway Rep multi graph ne g
Phylo parse handle
Nexus chain is parent of parent grandchild
HMM hidden markov model transitions from state_letter
core unknown seq back transcribe
Graphics comparative scatter plot draw to file output_file title
SeqIO insdc writer split multi line text max_len
GA Selection roulette wheel selection select population
core sqlite many seq files dict get raw key
Nexus chain get id
core feature location iter
Graphics GenomeDiagram feature set get ids
phenotype plate record len
Phylo tree format format_spec
Blast blast parser end parameters gap extend
Restriction not defined is unknown cls
codonalign get codon fold codon_table
core tagging consumer init handle colwidth maxwidth
PopGen GenePop file record remove locus by name name fname
NeuralNetwork Gene simple finisher init num_schemas min_generations
GA Repair ambiguous repair repair organism
Graphics GenomeDiagram track new set type
Graphics distribution page draw distributions cur_drawing start_x_pos x_pos_change start_y_pos
core safe peekline handle
PDB neighbor search init atom_list bucket_size
NeuralNetwork Gene schema find matches motif query
SubsMat seq mat full to half
ExPASy parse handle
core classify me observation
motifs write motifs format
AlignIO relaxed phylip iterator split id line
SearchIO HmmerIO hmmer2text parser parse hits
ExPASy parse handle
PDB residue depth residue surface
NeuralNetwork BackPropagation input layer init num_nodes next_layer
PDB fragment init length fid
Graphics bar chart distribution draw title cur_drawing title start_x start_y
Pathway network interactions
SearchIO _model query result index hit_key
Restriction restriction batch current suppliers
PDB entity set parent entity
PDB nice case line
PDB ppbuilder build peptides entity aa_only
Nexus tree convert absolute support nrep
SeqUtils seq1 seq custom_map undef_code
Phylo parse file format
Pathway system repr
Graphics GenomeDiagram circular drawer draw feature feature
Phylo PAML baseml init alignment tree working_dir out_file
Align multiple seq alignment get alignment length
PopGen GenePop gene pop controller calc fst all fname
Graphics GenomeDiagram circular drawer draw line graph graph
KEGG KGML entry is reactant
Graphics GenomeDiagram circular drawer draw feature set set
core split virtual offset virtual_offset
SeqIO fastq generic2 in_handle out_handle mapping truncate_char
Entrez efetch db
Pathway network str
SearchIO set hsp seqs hsp parsed program
PDB get atom radius atom rtype
GenBank base gen bank consumer convert to python numbers start end
Nexus tree search taxon taxon
Graphics spring layout desired minimum maximum gap
Phylo equal topology tree1 tree2
PDB calc angle v1 v2 v3
PDB structure alignment get iterator
core read and call uhandle method
Phylo PAML parse siteclass omegas line site_classes
Entrez open cgi params post ecitmatch
GenBank reference consrtm line
core indexed seq file dict close
Blast blast parser end statistics lambda
Restriction print format make list only ls title nc s1
codonalign count diff yn00 codon1 codon2 P codon_lst
SeqIO fasta writer write record record
Restriction commercially available suppliers cls
Graphics chromosome segment draw segment cur_drawing
Restriction ov5 overhang cls
PopGen GenePop gene pop commandline set input fname
KEGG KGML pathway add relation relation
Blast blast parser end hsp identity
Restriction analysis outside start end dct
Restriction blunt is 3overhang cls
Restriction ambiguous is defined cls
Phylo PAML parse model lines results
Restriction two cuts cut twice cls
Pathway Rep graph ne g
GenBank record organism line
Graphics comparative scatter plot find min max info
SearchIO ExonerateIO exonerate cigar indexer get qresult id pos
Phylo convert in_file in_format out_file out_format
SCOP parse handle
Alphabet get base alphabet alphabet
Nexus safename name mrbayes
Restriction print format print as what
GenBank read handle
Restriction analysis boundaries start end
NeuralNetwork Gene pattern io write seq seq_pattern_list output_handle
PDB entity get level
HMM calculate emissions emission_probs
Restriction formatted seq to linear
Restriction check bases seq_string
GenBank insdc scanner parse cds features handle alphabet tags2id
PDB vector angle other
SeqIO gen bank writer write single line tag text
KEGG Compound record str
SCOP parse handle
GenBank feature consumer locus locus_name
codonalign cal dn ds codon_seq1 codon_seq2 method codon_table
Phylo PAML paml print options
Nexus nexus insert gap pos n leftgreedy
SeqIO check eof handle index_offset index_length
AlignIO alignment writer clean text
PDB structure builder init residue resname field resseq icode
Blast ncbibaseblast commandline validate incompatibilities incompatibles
Graphics kgmlcanvas add compounds
Medline read handle
core seq getitem index
Phylo branch color str
GenBank feature consumer version suffix version
GA Mutation safe fitness mutation mutate org
Nexus unique label previous_labels label
Graphics GenomeDiagram diagram new track track_level
GenBank feature parser parse handle
Restriction ambiguous drop cls
Graphics organism draw legend cur_drawing start_y width
Phylo distance tree constructor nj distance_matrix
core one of position position
Pathway Rep multi graph edges label
Restriction formatted seq finditer pattern size
SearchIO HmmerIO hmmer3domtab hmmhit writer build row qresult
core feature location start
Nexus consensus trees threshold outgroup
SeqIO abi trim iterator handle
Compass read handle
core match all string trie
AlignIO maf index region2bin start end
Blast blast parser end hsp hit to
GA Crossover general point crossover init points crossover_prob
Phylo clade getitem index
Restriction restriction batch add nocheck other
core seq record bool
GA organism ne other
SeqUtils protein analysis aromaticity
Application abstract commandline get parameter name
Graphics GenomeDiagram circular drawer draw sigil jaggy bottom center top startangle
Restriction restriction batch add other
SCOP node get descendents node_type
core mutable seq le other
PDB atom get siguij
Pathway interaction str
Alphabet ungap alphabet
SeqIO genbank convert fasta in_handle out_handle alphabet
phenotype fit function x y
SCOP domain to des record
PDB select accept chain chain
Graphics chromosome component add component
Phylo writer write tree clade tree parent rooted
Blast blast parser end statistics eff space
SeqIO make position location_string offset
SCOP node get ascendent node_type
SearchIO ExonerateIO adjust aa seq fraglist
AlignIO maf index check existing db
FSSP mult align sum_dict align_dict
SCOP scop write des sql handle
SeqUtils codon adaptation index generate index fasta_file
SearchIO blat psl indexer get raw offset
SearchIO _model hspfragment set seq seq seq_type
GA Selection tournament selection select population
Graphics GenomeDiagram feature init parent feature_id feature color
Graphics kgmlcanvas add labels graphics
Restriction ov5 catalyse cls dna linear
PDB fragment mapper contains res
core mutable seq count sub start end
SearchIO BlastIO blast tab parser parse comments
Application abstract commandline check value value name check_function
PDB mmtf structure decoder finalize structure
Blast blast parser end blast output query def
phenotype well record repr
PopGen GenePop record split in loci gp
Alphabet has stop codon lower
Phylo matrix str
core test
SVDSuperimposer svdsuperimposer rms coords1 coords2
NeuralNetwork Gene simple finisher is finished organisms
Restriction two cuts modify cls location
Alphabet alphabet lower
motifs position specific scoring matrix mean background
Nexus node set data data
SearchIO convert in_file in_format out_file out_format
KEGG KGML entry str
SCOP astral get astral domains from file filename file_handle
SeqIO pdb seqres iterator handle
Restriction print format make number only ls title nc s1
core compound location nofuzzy start
SearchIO BlastIO blast tab parser parse fields line
Phylo tree mixin depths unit_branch_lengths
Restriction ov3 is 3overhang cls
SeqIO seq xml iterator elem dnaseq node record
SeqIO abi parse header header handle
PDB vector normalized
Align summary info gap consensus threshold ambiguous consensus_alpha require_multiple
PDB abstract property map keys
codonalign align shift recs recs
PDB vector normsq
Cluster record kcluster nclusters transpose npass method
Graphics GenomeDiagram diagram init name format pagesize orientation
Graphics GenomeDiagram circular drawer draw heat graph graph
SearchIO HmmerIO hmmer3tab parser parse qresult
Graphics GenomeDiagram graph data init id data name style
phenotype to opm plate
Restriction abstract cut all suppliers cls
GenBank feature consumer features line content
Phylo function matcher matcher_func
Graphics telomere segment draw segment cur_drawing
Cluster record init handle
SearchIO HmmerIO hmmer3tab writer build row qresult
Nexus char buffer next word
SearchIO HmmerIO hmmer2text parser parse key value
PDB QCPSuperimposer qcpsuperimposer get rotran
core bgzf writer isatty
SearchIO ExonerateIO get row dict row_len model
Phylo parse handle
Phylo tree as phyloxml
Phylo check str text testfunc
Restriction no cut cut twice cls
Restriction no cut cut once cls
core after position extension
PDB polypeptide get theta list
Restriction analysis test reverse start end site
Phylo tree element repr
core attempt read and call uhandle method
SeqUtils tm wallace seq check strict
SearchIO blat psl parser parse qresult
HMM hidden markov model get blank transitions
Restriction no cut modify cls location
AlignIO stockholm iterator populate meta data identifier record
NeuralNetwork BackPropagation output layer update previous_layer
KDTree kdtree all get radii
PopGen SimCoal sim coal cache get simulation sim_name ploidy parDir
KEGG KGML pathway add entry entry
motifs parse handle format
GA generation evolver init starting_population selector
Restriction abstract cut is isoschizomer cls other
Nexus node remove succ id
Phylo branch color to hex
SearchIO _model query result pop hit_key default
codonalign get codon rec pro nucl span_mode alphabet
Phylo tree mixin distance target1 target2
SeqIO phred quality from solexa solexa_quality
SeqIO string as base 36 string
