SearchIO ExonerateIO clean blocks tmp_seq_blocks
Restriction two cuts rev modify cls location
Nexus node get succ
Phylo get children text parent tag construct
PDB entity getitem id
SeqUtils nt search seq subseq
SeqIO fastq illumina writer write record record
Graphics GenomeDiagram circular drawer draw arc inner_radius outer_radius startangle endangle
Alphabet alphabet encoder lower
SeqIO write sequences handle format
SeqIO insdc writer write feature feature record_length
core seq record len
SearchIO HmmerIO hmmer3tab parser parse row
Blast blast parser end parameters matrix
Restriction restriction batch add supplier letter
motifs position specific scoring matrix dist pearson other
HMM abstract dpalgorithms backward recursion cur_state sequence_pos forward_vars
Blast blast parser end statistics entropy
SearchIO BlastIO blast tab parser get id parsed
PopGen GenePop file record start read
core reverse complement sequence
Cluster read handle
PDB pdbparser get header header_coords_trailer
Phylo clade to phylogeny
Phylo tree mixin find clades target terminal order
Restriction commercially available is comm cls
Phylo distance tree constructor height of clade
PDB chop end misc line
PDB atom get name
GenBank record definition line
SeqIO qual phred iterator handle alphabet title2ids
KEGG kegg info database
core translate sequence table stop_symbol to_stop
core sqlite many seq files dict load index
SearchIO _model hit description all
SCOP scop write hie sql handle
Phylo tree mixin filter search filter_func order follow_attrs
Pathway interaction repr
SeqUtils protein analysis protein scale param_dict window edge
core seq feature contains value
SeqIO seq xml writer write description record
Restriction commercially available buffers cls supplier
AlignIO maf writer write record record
SeqIO get read xy read_name
UniProt gpi11iterator handle
core indexed seq file dict values
Graphics GenomeDiagram diagram get levels
HMM known state trainer count emissions training_seq emission_counts
Entrez esummary
Phylo PAML parse ng86 lines results
SearchIO index filename format key_function
Phylo combine args first
Data make back table table default_stop_codon
Blast blast parser end hit def
Graphics chromosome segment draw cur_drawing
SCOP astral is domain in id dom id
Pathway system init reactions
SearchIO BlastIO blast xml writer write qresults qresults
KEGG KGML pathway element
PDB hsexposure ca get cb r1 r2 r3
PDB pdbparser get structure id file
Align summary info get gap char
Sequencing read handle
SearchIO BlastIO blast xml writer write file qresults
core find states markov_model output
Phylo PAML parse pairwise lines results
motifs position specific scoring matrix max
TogoWS entry db id format field
SeqUtils codon adaptation index set cai index index
GenBank gen bank scanner feed first line consumer line
GenBank insdc scanner parse handle do_features
Phylo branch color repr
PopGen GenePop read handle
Phylo PAML parse rates lines parameters
Blast blast parser end hsp query to
NeuralNetwork example manager add examples training_examples
Align multiple seq alignment extend records
AlignIO write alignments handle format
PDB structure builder set siguij siguij_array
SearchIO BlastIO compute gapopen num hsp
Restriction one cut cut twice cls
GenBank feature consumer reference num content
Graphics kgmlcanvas add maps
Blast blast parser end blast output query id
Graphics chromosome counts fill chromosome chromosome
core seq feature flip length
Phylo PAML parse yn00 lines results sequences
SeqIO sff writer init handle index xml
PDB fragment repr
core after position repr
core calc empirical expects xs ys classes features
Entrez data handler external entity ref handler context base systemId publicId
Emboss read handle
GA Selection diversity selection init internal_selector genome_generator
Graphics GenomeDiagram color translator read colorscheme filename
SearchIO _model hit fragments
Phylo sub clade clade term_names
core classify knn x weight_fn distance_fn
motifs jaspar jaspar5 fetch motif by id id
SearchIO BlastIO blast xml generator end parents num
NeuralNetwork Gene differential schema fitness init positive_seqs negative_seqs schema_evaluator
Restriction analysis with n sites N dct
HMM abstract dpalgorithms forward algorithm
Restriction print format make nocut only nc s1 ls title
HMM hidden markov model transitions to state_letter
Nexus nexus constant matrix delete exclude
Nexus quotestrip word
Align summary info get column info content obs_freq e_freq_table log_base random_expected
Phylo writer write handle cdao_to_obo
PDB chain getitem id
Restriction ov3 is blunt cls
core seq endswith suffix start end
motifs motif anticonsensus
GenBank feature consumer sequence content
NMR peaklist residue dict index
core bgzf reader iter
SeqIO fastq solexa convert fastq solexa in_handle out_handle alphabet
PDB vector to axis line point
NeuralNetwork Gene pattern repository remove poly a at_percentage
NeuralNetwork BackPropagation abstract layer set weight this_node next_node value
Restriction print format make list ls title nc s1
GenBank feature consumer topology topology
KEGG KGML entry add graphics entry
Graphics annotated chromosome segment overdraw subcomponents cur_drawing
Restriction defined mod2 cls other
core reference str
NMR predict noe peaklist originNuc detectedNuc originResNum
core feature location str
Cluster record somcluster transpose nxgrid nygrid inittau
SeqIO seq xml iterator elem aaseq node record
PopGen GenePop gene pop commandline set menu option_list
SeqIO embl writer write record record
PDB format date pdb_date
SearchIO ExonerateIO exonerate text indexer get qresult id pos
NeuralNetwork Gene pattern repository get random num_patterns
SearchIO _model query result append hit
Blast psiblast parser parse handle
Nexus nexus crop matrix matrix delete exclude
SVDSuperimposer svdsuperimposer run
PDB polypeptide repr
core run doctest target_dir
Graphics GenomeDiagram circular drawer draw bar graph graph
PDB fragment sub other
SeqUtils protein analysis count amino acids
PDB disordered entity wrapper disordered get id list
Graphics distribution page draw output_file title
Restriction restriction type truediv cls other
AlignIO clustal writer write alignment alignment
PDB pdblist init server pdb obsolete_pdb
SCOP astral get seq domain
core seq tomutable
SeqIO fastq generic in_handle out_handle mapping
PopGen GenePop gene pop controller calc ibd diplo fname stat scale min_dist
Phylo identity matcher target
phenotype well record str
Graphics GenomeDiagram color translator scheme color value
PDB disordered entity wrapper is disordered
motifs jaspar jaspar5 fetch internal id list collection tf_name tf_class tf_family
HMM training sequence init emissions state_path
core mutable seq repr
GA Crossover safe fitness crossover init actual_crossover accept_less
Nexus tree is bifurcating node
SearchIO _model query result init hits id hit_key_function
GenBank reference pubmed line
ExPASy get prosite raw id cgi
Nexus tree branchlength2support
AlignIO maf writer write header
HMM known state trainer count transitions state_seq transition_counts
Align summary info get all letters
Graphics chromosome counts color from count count
SearchIO BlastIO blast tab indexer qresult index commented
Restriction restriction type floordiv cls other
Restriction restriction batch iadd other
PDB disordered entity wrapper get id
SeqIO seq xml writer write properties record
Phylo PAML paml get option option
KEGG KGML reaction add substrate substrate_id
SearchIO BlastIO blast tab indexer get raw qresult offset
PopGen GenePop record remove locus by name name
ExPASy read handle
SeqUtils protein analysis instability index
Phylo PAML read results_file
Nexus standard data raw
KEGG KGML relation element
Application abstract commandline call stdin stdout stderr cwd
codonalign chisqprob x df
Restriction blunt is blunt cls
GA organism recalculate fitness
SearchIO BlastIO blast tab parser parse result row
AlignIO extract alignment region alignment_seq_with_flanking annotation
NeuralNetwork validation increase stop init max_iterations min_iterations verbose
PDB psea pname
core feature location flip length
Entrez espell
Phylo ns tag namespace
Graphics GenomeDiagram circular drawer draw tick tickpos ctr ticklen track
core bgzf writer close
SeqUtils molecular weight seq seq_type double_stranded circular
PopGen GenePop file record remove loci by position positions fname
core sqlite many seq files dict getitem key
SubsMat build subs mat obs_freq_mat exp_freq_mat
SearchIO fragcascade attr seq_type doc
Phylo PAML incomplete gamma x alpha
AlignIO maf index maf indexer
SeqIO insdc feature position string pos offset
Emboss read handle
core safe readline handle
PDB mmtf structure decoder set group info group_name group_number insertion_code group_type
HMM markov model builder set emission score seq_state emission_state probability
Nexus tree get taxa node_id
Graphics GenomeDiagram graph set get ids
Data codon table str
motifs instances search sequence
Graphics GenomeDiagram circular drawer canvas angle base
PDB entity has id id
GenBank record consumer features line content
Blast blast parser end blast output version
PopGen SimCoal get chr template stream chrs
core compound location flip length
UniProt gpa10iterator handle
core unknown seq init length alphabet character
GenBank record parser parse handle
Phylo Applications is numeric x
SCOP scop write hie handle
Pathway Rep multi graph child edges parent
Graphics GenomeDiagram feature set init set_id name parent
Pathway Rep multi graph add node node
codonalign count site yn00 codon_lst1 codon_lst2 pi k
PDB get surface model PDB_TO_XYZR MSMS
Restriction analysis between start end dct
core seq translate table stop_symbol to_stop cds
AlignIO alignment iterator iter
SeqIO xmlrecord iterator iter
PDB entity get parent
Graphics GenomeDiagram circular drawer draw
SearchIO fasta m10parser parse preamble
Graphics GenomeDiagram circular drawer draw sigil big arrow bottom center top startangle
phenotype well record get raw
NeuralNetwork BackPropagation logistic function value
SearchIO fasta m10indexer get raw offset
core calc model expects xs classes features alphas
NeuralNetwork Gene most count schema fitness init seq_records schema_evaluator
core before position extension
Pathway Rep multi graph eq g
PDB structure builder init chain chain_id
Blast blast parser end hit id
SearchIO ExonerateIO exonerate vulgar indexer get qresult id pos
Nexus nexus terminal gap to missing missing skip_n
Pathway system str
core compound location ref db
Restriction restriction type ne cls other
motifs motif str masked
SearchIO BlastIO blast xml generator start element name attrs children
PDB disordered entity wrapper detach parent
codonalign prim G
core mutable seq lt other
SearchIO _model base hsp str hsp header
NeuralNetwork Gene motif finder get motif dict seq_records motif_size
GenBank record str
SeqUtils tm nn seq check strict c_seq
Blast iterator init handle parser
Graphics GenomeDiagram circular drawer draw test tracks
SearchIO BlastIO blast tab writer adjust coords field value hsp
PDB fragment len
Phylo clade len
Graphics GenomeDiagram color translator artemis color value
GA organism init genome fitness_calculator start_fitness
PDB disordered entity wrapper disordered has id id
SearchIO write qresults handle format
Graphics GenomeDiagram feature set set all features attr value
Phylo bit string contains other
Nexus tree str
Alphabet gapped lower
Phylo PAML parse parameter list lines parameters num_params
Nexus chain all ids
GenBank feature consumer location content
SCOP seq map index getitem key
NeuralNetwork Gene pattern repository count pattern
AlignIO parse handle format seq_count alphabet
core indexed seq file dict iter
Graphics comparative scatter plot set colors and shapes scatter_plot display_info
Nexus node init data
Cluster record treecluster transpose method dist
Affy read handle
Blast iterator next
SCOP parse handle
SeqIO get read region read_name
SeqIO parser parse
core seq record repr
SCOP node get parent
Nexus tree is compatible tree2 threshold strict
Restriction analysis with number list list dct
Graphics GenomeDiagram abstract drawer is in bounds value
Compass record hit coverage
SeqIO seq xml iterator attr species attr_dict record
SeqIO fastq illumina convert fastq sanger in_handle out_handle alphabet
Restriction unknown catalyse cls dna linear
KEGG Enzyme parse handle
ExPASy get sprot raw id
Blast blast parser end parameters gap open
Phylo tree mixin prune target
NeuralNetwork Gene motif finder find seq_records motif_size
Graphics GenomeDiagram diagram write filename output dpi
Pathway network source species
Phylo PAML parse kappas lines parameters
PDB vector left multiply matrix
SeqUtils protein analysis molecular weight
PDB select accept atom atom
PopGen FDist convert genepop to fdist big old gp_rec report_loci
Phylo write obj file encoding indent
Restriction restriction type lt cls other
Phylo read file format
SeqIO seq xml writer write record record
core train training_set results feature_fns update_fn
Restriction analysis only outside start end dct
PDB min dist coord surface
KEGG Enzyme record init
SearchIO _model hit validate hsp hsp
core seq feature len
HMM markov model builder set initial probabilities initial_prob
Graphics GenomeDiagram stroke and fill colors color border
core seq ne other
SCOP astral domains clustered by id id
SeqIO seq xml iterator attr entry attr_dict record
Restriction restriction batch init first suppliers
Graphics darken color factor
GenBank feature consumer residue type type
SVDSuperimposer svdsuperimposer get rotran
SearchIO HmmerIO hmmer3domtab parser parse row
core restricted dict init length
Phylo phylogeny as phyloxml
Restriction ov5 is 3overhang cls
SeqIO seq xml iterator elem description node record
Nexus nexus get indices options set_type separator
GenBank feature consumer split reference locations location_string
GenBank reference remark line
Graphics color spiral init a b v_init v_final
Graphics GenomeDiagram track get sets
Phylo tree root with outgroup outgroup_targets
SCOP domain to cla record
SCOP astral get seq by sid domain
Crystal hetero ne other
core make virtual offset block_start_offset within_block_offset
motifs jaspar read handle format
SCOP record process line
SCOP index init filename
Restriction ov5 mod1 cls other
GenBank record nid line
Restriction restriction type rtruediv cls other
Restriction unknown compatible end cls
PopGen FDist approximate fst desired_fst simulated_fst parameter_fst max_run_fst
SeqUtils seq3 seq custom_map undef_code
Restriction print format print that dct title s1
SeqIO sequential sequence writer write file records
core load handle
Blast multiple alignment to generic alphabet
phenotype plate record add plate
Pathway Rep bf search graph root
PDB atom get altloc
Graphics telomere segment init inverted
PDB pdblist get all entries
SearchIO _model hit sort key reverse in_place
core between position extension
phenotype richards x A u d
SearchIO HmmerIO hmmer3text parser create hits hit_attrs qid qdesc
SearchIO HmmerIO hmmer3domtab hmmhit writer write file qresults
phenotype csv iterator handle
phenotype plate record get row row
Align pssm init pssm
Phylo tree searcher search starting_tree alignment
codonalign codon alignment to multiple seq alignment
codonalign codons2re codons
Graphics chromosome component remove component
core sqlite many seq files dict get k d
